root/tag/MoveTag.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormRowTextLong from '../components/FormRowTextLong';
+import FormSubmit from '../components/FormSubmit';
+import TagLink from '../static/scripts/common/components/TagLink';
+import bracketed from '../static/scripts/common/utility/bracketed';
+
+import TagLayout from './TagLayout';
+
+type Props = {
+  +$c: CatalystContextT,
+  +form: FormT<{
+    +tags: ReadOnlyFieldT<string>,
+  }>,
+  +tag: TagT,
+};
+
+const MoveTag = ({
+  $c,
+  form,
+  tag,
+}: Props): React.Element<typeof TagLayout> | null => {
+  const user = $c.user;
+
+  return user ? (
+    <TagLayout
+      $c={$c}
+      page="delete"
+      tag={tag}
+      title={l('Change tag uses')}
+    >
+      <h2>{l('Change tag uses')}</h2>
+      <p>
+        {exp.l(
+          `This will change all your votes for tag “{tag}”
+           to the tag or tags you specify.`,
+          {tag: <TagLink tag={tag.name} />},
+        )}
+        {' '}
+        {bracketed(
+          <a
+            href={'/user/' + encodeURIComponent(user.name) +
+                  '/tag/' + encodeURIComponent(tag.name)}
+          >
+            {l('see list')}
+          </a>,
+        )}
+      </p>
+      <p>
+        {l(`If you want to replace the tag with several others,
+            enter them separated by commas.`)}
+      </p>
+      <form action={$c.req.uri} method="post">
+        <div className="half-width">
+          <FormRowTextLong
```

Comment:
This displays weirdly. :) The input is really long and on a separate line from the label, which is indented almost to the middle of the input.

Reply:
It displayed on one line to me, but it might be better now?